app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -161,7 +161,7 @@ def handle_standard_error(err, enrollment)
       **response_attributes,
       reason: 'Request exception',
       exception_class: err.class.to_s,
-      exception_message: err.message,
+      exception_message: err_message,
```

Comment:
This file should be reverted to match `origin/main` since there are no functional changes.

Reply:
@NavaTim Thank you!  I made the change. 🙏🏻

Reply:
Cool, PR is approved!